engines/python/setup/djl_python/properties_manager/lmi_dist_rb_properties.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+#
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file
+# except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
+# BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+import logging
+import os
+from enum import Enum
+from typing import Optional
+
+import torch
+from pydantic.class_validators import root_validator
+
+from djl_python.properties_manager.hf_properties import get_torch_dtype_from_str
```

Comment:
Do we use this mapper anywhere else? If so it might be a good idea to move it out of a specific property implementation into a utils file.

Reply:
Yes will do it in the next ones